src/Microsoft.ML.AutoML/Utils/SweepableParamAttributes.cs

```diff
@@ -145,7 +145,7 @@ public SweepableFloatParam(float min, float max, float stepSize = -1, int numSte
 
         public override void SetUsingValueText(string valueText)
         {
-            RawValue = float.Parse(valueText, CultureInfo.InvariantCulture);
+            RawValue = float.Parse(valueText);
```

Comment:
I would instead push the culture invariant value into the sweepable parameter:
https://github.com/dotnet/machinelearning/blob/b861b5d64841cbe0f2c866ee7586872aac450a51/src/Microsoft.ML.Sweeper/Parameters.cs#L140